DE/Game.po

```diff
@@ -2118,14 +2118,14 @@ msgstr "Es wird hauptsächlich zum Fällen von Bäumen verwendet und kann bei Be
 #: /Game/Blueprint/ACF/Tutorial/WBP_TutorialTip1.WBP_TutorialTip1_C:WidgetTree.TextBlock_1.Text
 msgctxt ",6BCC8EA647892C6DDF2E86BC4A37CA07"
 msgid "现在尝试捡起前面的武器，并把它们在不同的栏位进行交换"
-msgstr "Versuchen Sie nun, die vorherigen Waffen aufzuheben und in verschiedenen Slots auszutauschen"
+msgstr "Versuchen Sie nun, die Waffen vor Ihnen aufzuheben und sie in verschiedene Slots zu ziehen"
 
 #. Key:	6BDE3B1B4C6870EFB6DFB3AC23B4C08E
 #. SourceLocation:	/Game/Blueprint/ACF/Item/Weapon/Gun/ACFGunBase_sniper_SVD.Default__ACFGunBase_sniper_SVD_C.ItemInfo.Name
 #: /Game/Blueprint/ACF/Item/Weapon/Gun/ACFGunBase_sniper_SVD.Default__ACFGunBase_sniper_SVD_C.ItemInfo.Name
 msgctxt ",6BDE3B1B4C6870EFB6DFB3AC23B4C08E"
 msgid "SVD"
-msgstr "SVD Scharfschützengewehr"
+msgstr "SVD"
```

Comment:
Please dont change the Weapon Name. Weapon Names are fix from the Dev. See pinned post in Discord.